pages/Stats.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <div
+    id="stats-viewer"
+    class="mx-4 mt-4 flex h-[91vh] flex-col"
```

Comment:
```suggestion
    class="mx-4 mt-4 flex h-[91vh] flex-col space-y-1"
```

Reply:
The number is more important since it is a stats page. It is easier to read in the middle.